Core/src/main/java/com/plotsquared/core/plot/flag/types/ListFlag.java

```diff
@@ -43,8 +43,18 @@ public ListFlag(final List<V> valueList, final Caption category, final Caption d
     @Override
     public F merge(@NonNull List<V> newValue) {
         final List<V> mergedList = new ArrayList<>();
-        mergedList.addAll(getValue());
-        mergedList.addAll(newValue);
+        // If a server already used PS before this fix, we remove all present duplicates on an eventual merge
+        for (final V v : getValue()) {
+            if (!mergedList.contains(v)) {
+                mergedList.add(v);
+            }
+        }
+        // Only add new values if not already present from #getValue()
+        for (final V v : newValue) {
+            if (!mergedList.contains(v)) {
+                mergedList.add(v);
+            }
+        }
```

Comment:
Not a particular fan of this implementation. The order shouldn't need to be preserved (it isn't really preserved when merging anyway) so other collection implementations (Set) might be better for code conciseness.

Reply:
Updating fixed that